src/common/chat/Tokenizer.ts

```diff
@@ -114,6 +115,10 @@ export class Tokenizer {
 
 		return (this.msg.tokens = tokens);
 	}
+
+	private isValidLink(message: string): boolean {
+		return parse(message).isIcann ?? false;
```

Comment:
nit: namespace with default import

```suggestion
		return tldts.parse(message).isIcann ?? false;
```